src/learn/math/drill/helper-problems.js

```diff
@@ -1,15 +1,15 @@
 // Define the problems for math drill
 
-const special1 = [...Array(9).keys()].map((a) => [0, a + 1]);
-const special2 = [...Array(9).keys()].map((a) => [a + 1, 0]);
-const special3 = [...Array(10).keys()].map((a) => [a + 1, 0]);
-const special4 = [...Array(18).keys()].map((a) => [a + 1, a + 1]);
-const special5 = [...Array(9).keys()].map((a) => [1, a + 1]);
-const special6 = [...Array(9).keys()].map((a) => [a + 1, 1]); // [1...9, 1]
-const special7 = [...Array(9).keys()].map((a) => [a + 1, a + 1]); // [[1,1],...[9,9]]
+const special1 = [...Array(9).keys()].map(a => [0, a + 1]);
```

Comment:
I know that single parameter arrow functions allow for now parenthesis but I've setup the lint rules to always enforce parenthesis for all arrow functions in order to be consistent in style across the whole repo. I think that this is super easy and quick to fix by just running `npm run lintfix`

Reply:
Another point. If we decided to change this lint rule, we should do that as a single distinct and separate PR and not mix it with these upgrade changes.

Reply:
Before commit all these changes and open PR, I really forgot to run this command, but I just ran `npm run lintfix` and nothing was changed. Maybe there are new rules with the latest eslint version we have updated?

Reply:
Well, after some attempts, it worked!